Plugins~/Src/MeshSyncClient/Include/MeshSyncClient/msTextureManager.h

```diff
@@ -24,6 +24,8 @@ class TextureManager
     int addFile(const std::string& path, TextureType type);
     // thread safe
     int add(TexturePtr tex);
+    // thread safe
+    int addPackedImage(const std::string& name, const void* data, size_t size, TextureType type);
```

Comment:
An image that's packed into the blend file.

Reply:
This file is in the MeshSyncClient namespace, which is independent of Blender or any DCC tool. Can this be renamed to addImage, so that it leaves out any Blender specific terms?

Reply:
I've refactored it to use the other method and removed this one now.